notebooks/whisper-asr-genai/whisper-asr-genai.ipynb

```diff
@@ -61,17 +61,29 @@
   },
```

Comment:
<p>could ypou please remove this cell output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2466/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>